roles/scaffold_tests/tasks/is_file.yaml

```diff
@@ -0,0 +1,14 @@
+- name: Set a fact for the dest file name
+  set_fact:
+    dest_fname: "{{ file['path'][(role_path + '/files/')|length:]|replace('module_name', module) }}"
```

Comment:
@cidrblock can we avoid the `replace` by having `module` in the actual file path?